target/linux/ramips/image/mt7621.mk

```diff
@@ -222,6 +222,15 @@ define Device/buffalo_wsr-600dhp
 endef
 TARGET_DEVICES += buffalo_wsr-600dhp
 
+define Device/cudy_wr2100
+  $(Device/dsa-migration)
+  DEVICE_VENDOR := Cudy
+  DEVICE_MODEL := WR2100
+  IMAGE_SIZE := 16064k
```

Comment:
That's not 0xf80000 ...

Reply:
oh :deer: 

thanks for pointing it out!